tools/client/Cargo.toml

```diff
@@ -13,7 +20,7 @@ base64.workspace = true
 clap = { version = "4.0.29", features = ["derive"] }
 env_logger.workspace = true
 jwt-simple = "0.11.4"
-kbs_protocol = { git = "https://github.com/confidential-containers/guest-components", rev = "43f6832" }
+kbs_protocol = { git = "https://github.com/jialez0/guest-components", branch = "https" }
```

Comment:
Here, we are waiting https://github.com/confidential-containers/guest-components/pull/273, for now temporarily pointing to the branch of downstream.